src/pages/mesh/advanced/caching/cache-control-headers.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Cache-control headers
+description: Specifies how to use headers to limit and modify the cache for GET requests.
+keywords:
+  - API Mesh
+  - Extensibility
+  - Cache
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+# Cache-control headers
+
+When a browser or a GET request accesses a URL, the site's response headers typically include a `cache-control` header, which determines how long the site will allow its data to be cached. For example, a site could have the following response header:
```

Comment:
```suggestion
When a browser or a GET request accesses a URL, the site's response headers typically include a `cache-control` header, which determines how long the site will allow its data to be cached. For example, a site could have the following response header:
```

Maybe we need to update the wording here.

I don't know exactly how to word it but Instead of talking about a browser or a GET request accessing a GET request, talk about a HTTP Client such as a browser or `fetch` or a CDN.